src/eval-frame/iodide-evaluation-results.js

```diff
@@ -1,2 +1,5 @@
 export const IODIDE_EVALUATION_RESULTS = {};
+// FIXME: this should not be atttached to window --
+// in repInfoRequestResponse, we can set the environment to use the correct
+// root object rather than passing down rootObjName
```

Comment:
couldn't this in fact now be part of the redux store? the `IODIDE_EVALUATION_RESULTS` are serialized objs in some format, correct?

Reply:
oh nvm